pages/k8s/autoscaler.md

```diff
@@ -0,0 +1,140 @@
+---
+wrapper_template: "templates/docs/markdown.html"
+markdown_includes:
+  nav: "kubernetes/docs/shared/_side-navigation.md"
+context:
+  title: "Kubernetes autoscaler charm"
+  description: Deploy, configure and manage the cluster autoscaler
+keywords: charms, operators
+tags: [operating]
+sidebar: k8smain-sidebar
+permalink: autoscaler.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
+The Kubernetes autoscaler charm is an operator charm designed to be run on top
+of a **Charmed Kubernetes** cluster. The charm automatically adjusts
+the size of the Kubernetes cluster when one of the following conditions is true:
+
+ - there are pods that failed to run in the cluster due to insufficient resources
```

Comment:
failed to run due to insufficient resources is a bit vague here. Scale up occurs when pods are marked as unschedulable by the scheduler, so something that indicates its the failing to be scheduled that causes things would be be better I think
```suggestion
 - there are pods that failed to be scheduled in the cluster due to insufficient resources
```